modules/core/src/main/scala/ch/epfl/bluebrain/nexus/rdf/Decoder.scala

```diff
@@ -113,6 +114,17 @@ trait StandardDecoderInstances {
     }
   }
 
+  implicit final val graphDecodeDuration: Decoder[Duration] =
```

Comment:
Shouldn't be better than when decoding and fails we always print the original value?

`Unable to decode node with value '{value}' as a Duration`

This goes for actually all the `Decoder`